### PR TITLE
New implementation request

### PR DESCRIPTION
**Pull request by RamonGilabert**
It doesn't have to be merged, but we could leave it as a branch for now, look at it and tell me what you think, the code is much simpler in my opinion.

app

Next steps are:

- [ ] Get the initial frame of the image and make an animation with it.
- [ ] Add more customization points as for instance a description and stuff.

### CODE REVIEW

**Comment by RamonGilabert** [comment]
Also it improves the animations that were sort of broken in the other part, and the transition. @hyperoslo/ios

**Comment by zenangst** [comment]
I think this looks good, looking forward to see it finished.
Like the deletion count contra the added :top:

File: Source/LightboxTransition.swift [hunk 1]
```diff
@@ -1,240 +1,146 @@
 import UIKit
 
-protocol LightboxTransitionDelegate: class {
+public class LightboxTransition: UIPercentDrivenInteractiveTransition {
```
Comment (vadymmarkov) [inline Source/LightboxTransition.swift]: Nice.

**Comment by vadymmarkov** [comment]
Looks good for me, much simpler. @RamonGilabert I think if we add rotation support then we have all the features as before, so it could be merged.

**Comment by zenangst** [comment]
@vadymmarkov @RamonGilabert is this good to go?

**Comment by vadymmarkov** [comment]
@zenangst No, there is one more thing with rotation, like anchor point.

**Comment by RamonGilabert** [comment]
# I think so.

How cool is that? You can add **stuffs into** _comments_ now.

> That's what I said.

**Comment by zenangst** [comment]
@vadymmarkov AH! that rotation.

**Comment by zenangst** [comment]
@RamonGilabert @vadymmarkov :trophy:

**Comment by RamonGilabert** [comment]
Crazy collaboration. 🤗

**Pull request merged**